RELEASING.md

```diff
@@ -246,25 +222,20 @@ $ git push
 
 Each time new commits are pushed, a new release candidate will be built by the CI.
 
-#### Tag The Final Release
+#### Tag and Publish the Final Release
 
 When QA are satisfied that a build is working, it's time to create the final
-tag.
-
-* If you're releasing SimplyE, use `simplye-99.0.0` as the tag name.
-* If you're releasing Open eBooks, use `openebooks-99.0.0` as the tag name.
-
-In order to avoid writing the same instructions twice, we'll refer to whichever
-tag you're using as `$app-99.0.0`. You'll obviously need to substitute in the
-correct tag name yourself.
+tag, and publish releases to GitHub and the Play Store. This is automated by the
+Create Release workflow in GitHub.
 
-```
-$ git tag -s '$app-99.0.0'
-$ git push --tags
-```
+1. In GitHub, open the Actions tab, and select the "Create Release" workflow.
+2. Open the "Run workflow" dropdown.
+3. In the "Use workflow from" dropdown, select the `release/99.0.0` branch.
+4. Click the "Run workflow" button.
 
-When the CI encounters a tagged commit, it will push the build it produces to
-the Play Store, and will push various components to Maven Central.
+The workflow will tag the commit, and create a GitHub release with generated release
+notes. It produce a release build, and push it to the Play Store, with a link to the
```

Comment:
Isn't there a typo in "It produce"? I was thinking of "It produces" or "It will produce", maybe.

Reply:
Yes, thank you!